### PR TITLE
Adding ADC declarations for B-L072Z-LORAWAN1 board

### PR DESCRIPTION
**Pull request by ichatz**
### Contribution description

Updated the boards/b-l072z-lrwan1/ directory to define the ADC wiring.

### Testing procedure

Simply use the adctest with the A0 or A2 ardiuono connectors.

### CODE REVIEW

File: boards/b-l072z-lrwan1/Makefile.features [hunk 1]
```diff
@@ -2,6 +2,7 @@ CPU = stm32
 CPU_MODEL = stm32l072cz
 
 # Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_adc
```
Comment (benpicco) [inline boards/b-l072z-lrwan1/Makefile.features:5]: `boards/b-l072z-lrwan1/Kconfig` will also need an update

Comment (ichatz) [reply boards/b-l072z-lrwan1/Makefile.features:5]: I made an additional push request. Sorry for missing this out.

**Comment by benpicco** [comment]
Thank you!
You can squash both commits into one, please use the board as a prefix in the commit message, something like

    boards/b-l072z-lrwan1: add ADC definitions

to keep then git log tidy.

**Comment by aabadie** [comment]
Any problem with this PR @ichatz ?

**Pull request closed without merging**